bindings/wasm/package.json

```diff
@@ -19,7 +19,7 @@
     ],
     "scripts": {
         "build": "npm run build:nodejs && npm run build:web",
-        "build:src": "cargo build --lib --release --target wasm32-unknown-unknown",
+        "build:src": "cargo build --lib --profile=production --target wasm32-unknown-unknown",
         "build:nodejs": "node ./build_scripts/copyNodejsDefs.js && npm run build:src && npm run bundle:nodejs && wasm-opt -O node/wasm/iota_sdk_wasm_bg.wasm -o node/wasm/iota_sdk_wasm_bg.wasm",
         "build:web": "node ./build_scripts/copyNodejsDefs.js && npm run build:src && npm run bundle:web && wasm-opt -O web/wasm/iota_sdk_wasm_bg.wasm -o web/wasm/iota_sdk_wasm_bg.wasm",
         "bundle:nodejs": "wasm-bindgen ../../target/wasm32-unknown-unknown/release/iota_sdk_wasm.wasm --typescript --weak-refs --target nodejs --out-dir node/wasm && node ./build_scripts/node && tsc --project tsconfig.node.json --outDir node/lib",
```

Comment:
There is `release` in a path here, is it related to the profile?

Reply:
Yes